docs/pages/dialogs.md

```diff
@@ -0,0 +1,95 @@
+# Dialogs
+
+This component uses [dialog element](https://www.w3.org/TR/2013/CR-html5-20130806/interactive-elements.html#the-dialog-element), which is only supported by Chrome and Opera currently. For other browsers, you need to include a [polyfill](https://github.com/GoogleChrome/dialog-polyfill) in your code.
+
+## Demo
+
+### Simple Dialog
+
+```jsx_demo_class
+class Demo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  openDialog() {
+    this.setState({
+      openDialog: true
+    });
+  }
+
+  closeDialog() {
+    this.setState({
+      openDialog: false
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <Button colored onClick={this.openDialog.bind(this)} raised ripple>Show Dialog</Button>
```

Comment:
Please rename the function `handleOpenDialog` and bind the function inside the constructor instead.
`this.handleOpenDialog = this.handleOpenDialog.bind(this);`
Same for the close event, in both examples :)